### PR TITLE
Desktop Release Notes for 1.42.x - Release

### LINKED ISSUE #24378: Desktop Release Notes for 1.42.x - Release
### Release Notes

 - Upgrade from Chromium 104.0.5112.69 to Chromium 104.0.5112.81. ([#24364](https://github.com/brave/brave-browser/issues/24364))
 - Farbled HTTP Accept-Language header does not have a q value. ([#24126](https://github.com/brave/brave-browser/issues/24126))
 - Remove `Get image description from Brave` context menu option. ([#24047](https://github.com/brave/brave-browser/issues/24047))
 - Browser crashes in SolanaProviderImpl::SelectedAccountChanged when trying to remove selected Ledger Solana account . ([#24046](https://github.com/brave/brave-browser/issues/24046))
 - Extraneous data in eth_call response is not handled correctly. ([#23998](https://github.com/brave/brave-browser/issues/23998))
 - [hackerone] wallet API freezing issue. ([#23973](https://github.com/brave/brave-browser/issues/23973))
 - Errors listing NFTs on OpenSea. ([#23889](https://github.com/brave/brave-browser/issues/23889))
 - Support using ledger hardware for Solana signTransaction, signAllTransactions provider APIs. ([#23842](https://github.com/brave/brave-browser/issues/23842))
 - Broken Image for Solana on localhost. ([#23821](https://github.com/brave/brave-browser/issues/23821))
 - Update search engine defaults for AT / ES / MX . ([#23814](https://github.com/brave/brave-browser/issues/23814))
 - Set macOS minimum version to 10.13 for 1.42.x in Sparkle. ([#23748](https://github.com/brave/brave-browser/issues/23748))
 - Add custom asset page not loading. ([#23694](https://github.com/brave/brave-browser/issues/23694))
 - [hackerone] IPFS crash. ([#23646](https://github.com/brave/brave-browser/issues/23646))
 - Add Loading Skeletons to Wallet Panel. ([#23577](https://github.com/brave/brave-browser/issues/23577))
 - [hackerone] Add warning that local ipfs node will bypass browser proxy. ([#23570](https://github.com/brave/brave-browser/issues/23570))
 - NEAR integration (Aurora). ([#23522](https://github.com/brave/brave-browser/issues/23522))
 - Pinned tabs, when muted, can't always be unmuted by clicking on unmute icon. ([#23476](https://github.com/brave/brave-browser/issues/23476))
 - Remove v1 shields UI (a.k.a. shields extension panel). ([#23285](https://github.com/brave/brave-browser/issues/23285))
 - Enable 1PES by default. ([#23180](https://github.com/brave/brave-browser/issues/23180))
 - Remove P3A protobuf reporting. ([#23147](https://github.com/brave/brave-browser/issues/23147))
 - Add ability to debounce URLs based on a regex applied to the path. ([#23121](https://github.com/brave/brave-browser/issues/23121))
 - Add setting to allow separate search engine usage for Private Window. ([#23091](https://github.com/brave/brave-browser/issues/23091))
 - Add new setting to show/hide bookmarks and history from the address bar autocomplete results . ([#22948](https://github.com/brave/brave-browser/issues/22948))
 - Wallet should use the user locale when displaying currency symbols. ([#22815](https://github.com/brave/brave-browser/issues/22815))
 - Fix cosmetic filters not being applied inside of child frames. ([#22781](https://github.com/brave/brave-browser/issues/22781))
 - Add localisation for tab-titles on add-switch chain dapps panel. ([#22672](https://github.com/brave/brave-browser/issues/22672))
 - Evmos actions failing (such as voting). ([#22650](https://github.com/brave/brave-browser/issues/22650))
 - Add link to support page within wallet. ([#22598](https://github.com/brave/brave-browser/issues/22598))
 - Refresh design for NTP Private Window. ([#22566](https://github.com/brave/brave-browser/issues/22566))
 - Bright flash of light upon opening a new tab. ([#16078](https://github.com/brave/brave-browser/issues/16078))
 - Add list of well known chains to the local data files updater. ([#14574](https://github.com/brave/brave-browser/issues/14574))
 - Import from Chromium version of Microsoft Edge. ([#13334](https://github.com/brave/brave-browser/issues/13334))
 - Filter out Marketo's email tracker from URLs. ([#9018](https://github.com/brave/brave-browser/issues/9018))
 - Delete icon in cookie settings does not delete some entries set from Shields or other settings. ([#8604](https://github.com/brave/brave-browser/issues/8604))

### PR DESCRIPTION
**Pull request by LaurenWags**
Fix #24378

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A
  - [ ] https://github.com/brave/brave-browser/wiki/Brave-Wallet

## Test Plan:

n/a

### CODE REVIEW

**Review by bsclifton — APPROVED** [review]
LGTM ++

**Review by rebron — APPROVED** [review]
++ Looks good.

File: CHANGELOG_DESKTOP.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## [1.42.86](https://github.com/brave/brave-browser/releases/tag/v1.42.86)
+
+ - Added Aurora EVM chain to preloaded chains for Brave Wallet. ([#23522](https://github.com/brave/brave-browser/issues/23522))
+ - Added loading skeletons to Brave Wallet panel. ([#23577](https://github.com/brave/brave-browser/issues/23577))
+ - Added support for using Ledger hardware for Solana signTransaction, signAllTransactions provider APIs with Brave Wallet. ([#23842](https://github.com/brave/brave-browser/issues/23842))
```
Comment (srirambv) [inline CHANGELOG_DESKTOP.md:7]: Probably can update to `Added support for using Ledger hardware for Solana Sign Transactions using Brave wallet`

File: CHANGELOG_DESKTOP.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+ - [Security] Fixed wallet API freezing issue as reported on HackerOne by jordaaash. ([#23973](https://github.com/brave/brave-browser/issues/23973))
+ - Enabled "unlinkable bouncing" by default. ([#23180](https://github.com/brave/brave-browser/issues/23180))
+ - Improved fingerprint farbling in aggressive mode. ([#24126](https://github.com/brave/brave-browser/issues/24126))
+ - Updated default search engine to Brave Search for new installations in certain regions. ([#23814](https://github.com/brave/brave-browser/issues/23814))
+ - Updated macOS minimum version to 10.13 in Sparkle. ([#23748](https://github.com/brave/brave-browser/issues/23748))
+ - Updated New Tab page design for private window and private window with Tor. ([#22566](https://github.com/brave/brave-browser/issues/22566))
+ - Updated brave://settings/cookies to better align with brave://shields. ([#8604](https://github.com/brave/brave-browser/issues/8604))
+ - Updated eth_call in Brave Wallet to handle extraneous data correctly. ([#23998](https://github.com/brave/brave-browser/issues/23998))
```
Comment (srirambv) [inline CHANGELOG_DESKTOP.md:23]: Probably change to `Fixed incorrect balance being shown for certain tokens`.

File: CHANGELOG_DESKTOP.md (hunk 1 above)
Comment (LaurenWags) [reply CHANGELOG_DESKTOP.md]: Updated

File: CHANGELOG_DESKTOP.md (hunk 2 above)
Comment (LaurenWags) [reply CHANGELOG_DESKTOP.md]: Updated and added "in Brave Wallet" to the end of suggested text.

**Review by srirambv — APPROVED** [review]
++

**Pull request merged**